react/week2/todolist/src/components/Todos.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+function AddTodo() {
```

Comment:
It's a good practice to keep the naming consistent, so since you named your file Todos.js and this component contains your Todos, I would rename AddTodo to Todos.
It also keeps it consistent with the logic inside the component :)